packages/schemas/tables/_after_all.sql

```diff
@@ -28,6 +28,11 @@ revoke all privileges
   on table systems
   from logto_tenant_${database};
 
+---- Revoke all privileges on service_logs table for tenant roles ----
+revoke all privileges
```

Comment:
Should AdminTenent admin user role be able to access this log table?

Reply:
No, only cloud (db super admin) can access this table